src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -460,5 +463,43 @@ public void ReleaseSubscriptionsSemaphore()
         {
             _concurrentConnectionsSemiSemaphore.Release();
         }
+
+        internal void CleanupSubscriptions()
+        {
+            var maxTaskLifeTime = (PlatformDetails.Is32Bits || this._db.Configuration.Storage.ForceUsing32BitsPager
+                        ? TimeSpan.FromHours(12)
```

Comment:
I don't think it matters, to be frank. Either way is very long

Reply:
Yeah, I didn't want to add a new configuration here, so I decided to have a small preemptive move